doc_indexer/src/utils/settings.py

```diff
@@ -5,6 +5,8 @@
 from decouple import Config, RepositoryEnv, config
 from dotenv import find_dotenv, load_dotenv
 
+# TODO: re-use the settings parent project
```

Comment:
Is it todo suppose to be completed as part of this PR?

Reply:
it should be part of this [issue](https://github.com/kyma-project/kyma-companion/issues/243). Do you want me to add the issue link to the comment too?

Reply:
Please remove the todo comment from the code then. Having it in the issue is enough.